plugins/collaajUpdatePlugin/lib/api/KalturaCaptureSpaceUpdateResponse.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class KalturaCaptureSpaceUpdateResponse extends KalturaObject {

```

Comment:
@tan-tan-kanarek 
The XML we agreed on with Collaaj had nested nodes.
to make our API response output such an XML, we need an object within an object, and this is why we had to add a property which is of this "info" type.